server/rpc.go

```diff
@@ -228,7 +228,7 @@ func verifyToken(orch Orchestrator, creds string) (*lpTypes.Job, error) {
 	job, err := orch.GetJob(token.JobId)
 	if err != nil || job == nil {
 		glog.Error("Could not get job ", err)
-		return nil, fmt.Errorf("Missing job")
+		return nil, fmt.Errorf("Missing job (%s)", err.Error())
```

Comment:
Not important right now, but for the future a slightly shorter way to do this is
`fmt.Error("Missing job: ", err)`